src/SFA.DAS.Commitments.Application/Commands/UpdateApprenticeshipStatus/UpdateApprenticeshipStatusCommandHandler.cs

```diff
@@ -74,10 +74,12 @@ IAcademicYearValidator academicYearValidator
 
         private async Task CreateEvent(UpdateApprenticeshipStatusCommand command, Apprenticeship apprenticeship, Commitment commitment, PaymentStatus newPaymentStatus)
         {
-            var isResuming = (newPaymentStatus == PaymentStatus.Active && apprenticeship.PauseDate.HasValue);
+            var apprenticeshipHasStarted = apprenticeship.StartDate.Value.Date < _currentDate.Now.Date;
```

Comment:
There is a little extension method on Apprenticeship called IsWaitingToStart which you could use instead

Reply:
Yeah, I forgot about that.